cropper/src/test/java/com/canhub/cropper/BitmapUtilsTest.kt

```diff
@@ -0,0 +1,126 @@
+package com.canhub.cropper
+
+import io.mockk.mockkObject
+import io.mockk.unmockkObject
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+private val RECT_POINTS: FloatArray = floatArrayOf(1f,2f,3f,4f, 5f, 6f, 7f, 8f)
```

Comment:
Let's extract the magic numbers into consts, like `rectLeft = 1f` so we can understand what which one represents